src/modules/item/ItemScreen.tsx

```diff
@@ -20,25 +20,17 @@ const ItemScreen = () => (
   </ItemContextProvider>
 );
 
-ItemScreen.propTypes = {
-  match: PropTypes.shape({
-    params: PropTypes.shape({ itemId: PropTypes.string }).isRequired,
-  }).isRequired,
-};
-
-const WrappedItemScreen = () => {
+const WrappedItemScreen = (): JSX.Element => {
   const { mutate: signOut } = useMutation(MUTATION_KEYS.SIGN_OUT);
-  const { mutate: itemLoginSignIn } = useMutation(
-    MUTATION_KEYS.POST_ITEM_LOGIN,
-  );
+  const { mutate: itemLoginSignIn } = usePostItemLogin();
   const { rootId } = useParams();
 
   const ForbiddenContent = <ItemForbiddenScreen />;
 
   const Component = ItemLoginAuthorization({
     signIn: itemLoginSignIn,
     signOut,
-    itemId: rootId,
+    itemId: rootId || '',
```

Comment:
isn't null better? If it's too avoid changing ui you can add a comment with the better solution maybe.